client/components/notes-formatted-block/test/blocks.js

```diff
@@ -130,10 +142,14 @@ describe( 'Post block', () => {
 		};
 
 		const text = 'another post';
-		const post = shallow( <Blocks.Post content={ content } children={ text } /> );
-
-		expect( post.type() ).toEqual( 'em' );
-		expect( post.text() ).toEqual( text );
+		const unlinkedPost = render( <Blocks.Post content={ content } children={ text } /> ).container
+			.firstChild;
+
+		// @todo I'm not sure about this assertion. I think it is important
```

Comment:
Feel free to keep this as you did it and remove the comment. Rationale makes sense to me. We might have trouble if we enable certain `jest-dom` ESLint rules, but for now, we're good, as this follows the style of the rest of the tests we have.